src/browser/browser.ts

```diff
@@ -588,3 +556,55 @@ export class Browser {
     }, zoomLevel);
   }
 }
+
+declare global {
+  interface Window {
+    __grafanaSceneContext: object;
+    __grafanaRunningQueryCount?: number;
+  }
+}
+
+async function waitForQueriesAndVisualizations(page: puppeteer.Page, options: ImageRenderOptions) {
+  await page.waitForFunction(
+    (isFullPage) => {
+      if (window.__grafanaSceneContext) {
+        return window.__grafanaRunningQueryCount === 0;
+      }
```

Comment:
@AgnesToulet pushed an update that restores the old waiting logic, except for scenes it will use this window query counter (That is not related to network query state but actual data source requests observable state and responses).  So for Athena this will work as they are returning observable loading states (And their observable does not complete until they they are done)

Reply:
@AgnesToulet it works for every slow query I have tested for any data source, in both new and old dashboard arch. 

What version of Grafana are you using and what is the url? is the feature toggle __feature.dashboardSceneSolo enabled (or in url)?

Reply:
@torkelo I tried to access the associated `/d-solo` page, and I'd say the issue is from there. The panel using the TestData DS with the slow query scenario always shows the loading page (the query is supposed to run for 15 seconds, but even after that time, I still see the loading page), and I don't see any query pending. 

So, I will approve the PR and will work on releasing it along with the PDF option next week.

Reply:
@AgnesToulet strange, d-solo route works well for me no matter the panel or query

Reply:
@AgnesToulet you you using latest main?

Reply:
@torkelo Actually, I tried again with a simple slow panel, and it's working. The panel that doesn't work is a repeating panel (using a custom variable).

Reply:
@torkelo for repeated panels you need use a new syntax for the panelId , example `panel-2-clone-2` , it's one of the few breaking changes in scenes